miri/datamodels/tests/test_reset_switch_charge_decay_model.py

```diff
@@ -83,33 +61,18 @@ def test_referencefile(self):
         self.assertIsNotNone(pedigree)
 
     def test_creation(self):
-        # Check that the field names in the class variable are the same
-        # as the ones declared in the schema.
-        class_names = list(MiriResetSwitchChargeDecayModel.fieldnames)
-        schema_names = list(self.dataproduct.get_field_names('rscd_table'))
-        self.assertEqual(class_names, schema_names,
-                         "'fieldnames' class variable does not match schema")
-
         # It must be possible to create an empty data product and fill
         # in its contents later.
         nulldp = MiriResetSwitchChargeDecayModel( )
         descr1 = str(nulldp)
         self.assertIsNotNone(descr1)
-        nulldp.rscd_table = self.rscddata
-        self.assertIsNotNone(nulldp.rscd_table)
-        descr2 = str(nulldp)
-        self.assertIsNotNone(descr2)
-        del nulldp, descr1, descr2    
+        del nulldp, descr1
 
     def test_copy(self):
         # Test that a copy can be made of the data product.
         datacopy = self.dataproduct.copy()
         self.assertIsNotNone(datacopy.rscd_table)
```

Comment:
See comment for line 87 for context: this line 74 is the 2nd place where the same error occurs.